webapp/src/styles/_components.scss

```
@@ -1,5 +1,5 @@
 .height-rhs {
-    height: 100%;
+    height: calc(100% - $rhs-bottom-button-height); // 100% is the entire height and 150px is the space occupied by the button fixed at the bottom
```

```suggestion
    height: calc(100% - #{$rhs-bottom-button-height}); // 100% is the entire height and 150px is the space occupied by the button fixed at the bottom
```
I think `calc` won't work this way - `calc(100% - $rhs-bottom-button-height)`.